stream.go

```diff
@@ -136,6 +134,8 @@ func (s *Stream) Close() error {
 	s.closedLocal = true
 	s.clLock.Unlock()
 
+	err := s.mp.sendMsg((s.id<<3)|Close, nil, time.Time{})
+
```

Comment:
Yes, this is a problem. We can end up closing the connection before all closes are sent.